config/environments/production.rb

```diff
@@ -30,15 +30,12 @@
     'Expires' => 'Thu, 31 Dec 2037 23:55:55 GMT'
   }
 
-  # Compress JavaScript using a preprocessor
-  config.assets.js_compressor = :uglifier
-
-  # Compress CSS using a preprocessor.
-  config.assets.css_compressor = :sass
-
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
 
+  config.assets.js_compressor = :uglifier
+  config.assets.css_compressor = :sass
```

Comment:
I believe this change was just introduced to remove stock comments that the author didn't find helpful, however since this feature branch is very large, I'd like to decouple it from unrelated refactorings. I'm going to undo this change.

Reply:
I believe these config changes were introduced to remove stock comments, but since this PR is very large I'd like to decouple it from unrelated refactorings. I'm going to remove this change and the change to the staging environment.